src/chain/shared_state.rs

```diff
@@ -220,3 +230,55 @@ impl NonEmptyList<(SectionInfo, ProofSet)> {
         }
     }
 }
+
+#[derive(Clone, PartialEq, Eq)]
+pub struct SectionProofBlock {
+    key: PublicKey,
```

Comment:
Would it make sense as BlsPublicKey and BlsSignature?
I'm thinking having 2 different crypto keys/signature will cause confusion if we are not explicit.

Reply:
I was kind of aiming for a drop-in replacement of BLS. With these names, we can just replace `use bls_emu::...` with `use threshold_crypto::...` and it should (mostly) just work. I can `use bls_emu::PublicKey as BlsPublicKey` etc., though.

Reply:
Yes, could also do that at the crate level in lib.rs so you only do it once and then use crate::{BlsPublicKey, ...}